tickets/src/main/java/no/unit/nva/publication/ticket/FileApprovalThesisDto.java

```diff
@@ -0,0 +1,119 @@
+package no.unit.nva.publication.ticket;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Username;
+import no.unit.nva.model.associatedartifacts.file.File;
+import no.unit.nva.publication.model.business.FilesApprovalThesis;
+import no.unit.nva.publication.model.business.PublishingWorkflow;
+import no.unit.nva.publication.model.business.TicketEntry;
+import no.unit.nva.publication.model.business.User;
+import nva.commons.core.JacocoGenerated;
+
+public class FileApprovalThesisDto extends TicketDto {
+
+    private static final String WORKFLOW_FIELD = "workflow";
+    private static final String APPROVED_FILES_FIELD = "approvedFiles";
+    private static final String FILES_FOR_APPROVAL = "filesForApproval";
+
+    @JsonProperty(WORKFLOW_FIELD)
+    private final PublishingWorkflow workflow;
+    @JsonProperty(APPROVED_FILES_FIELD)
+    private final Set<File> approvedFiles;
+    @JsonProperty(FILES_FOR_APPROVAL)
+    private final Set<File> filesForApproval;
+
+    @SuppressWarnings({"PMD.ExcessiveParameterList"})
+    @JsonCreator
+    public FileApprovalThesisDto(@JsonProperty(STATUS_FIELD) TicketDtoStatus status,
+                                @JsonProperty(CREATED_DATE_FIELD) Instant createdDate,
+                                @JsonProperty(MODIFIED_DATE_FIELD) Instant modifiedDate,
+                                @JsonProperty(IDENTIFIER_FIELD) SortableIdentifier identifier,
+                                @JsonProperty(PUBLICATION_IDENTIFIER_FIELD) SortableIdentifier publicationIdentifier,
+                                @JsonProperty(ID_FIELD) URI id,
+                                @JsonProperty(MESSAGES_FIELD) List<MessageDto> messages,
+                                @JsonProperty(VIEWED_BY) Set<User> viewedBy,
+                                @JsonProperty(ASSIGNEE_FIELD) Username assignee,
+                                @JsonProperty(OWNER_FIELD) User owner,
+                                @JsonProperty(OWNER_AFFILIATION_FIELD) URI ownerAffiliation,
+                                @JsonProperty(WORKFLOW_FIELD) PublishingWorkflow workflow,
+                                @JsonProperty(APPROVED_FILES_FIELD) Set<File> approvedFiles,
```

Comment:
Weird that we use set in the signatures. Probably better to use Collection<File> and then coerce the collection into a HashSet. (I see that we have this same issue elsewhere in this constructor).